WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -220,6 +220,14 @@ protected SendControlViewModel(WalletService walletService, string title)
 					}
 				});
 
+			this.WhenAnyValue(x => x.Address, x => x.CustomChangeAddress)
+				.ObserveOn(RxApp.MainThreadScheduler)
```

Comment:
I wonder if this is potentially infinite-loop inducing still? Though now that i think about it, `this.RaiseAndSetIfChanged` will stop change notifications with no actual value changes anyway

Reply:
That was my concern too. But according the tests it is fine. I would feel better if there would be some evidence in some documentation that this is OK. 
Or some other method to trigger the validation.

Reply:
Not sure what tests are you referring to, as no code tests this, nor it could as the infinite loop is OS specific. I think @jmacato is right, this PR should be reverted to play it safe.